dataland-frontend/src/utils/Constants.ts

```diff
@@ -30,4 +30,7 @@ export const TIME_DELAY_BETWEEN_SUBMIT_AND_NEXT_ACTION_IN_MS = 2000;
 
 export const UPLOAD_FILE_SIZE_DISPLAY_DECIMALS = 2;
 
+// Same as standard for windows machines. File name can't have one of the special characters <>:"|?/\ and shouldn't end with a whitespace or .
+export const REGEX_FOR_FILE_NAMES = /^(?=.{1,254}$)[^<>:"|?/*\\]+[^<>:"|?/*\\.\s]$/;
```

Comment:
Whitespace in the beginning of a filename seems not to be allowed as well.

Reply:
Are you sure? They way I understand the regex, a whitespace at the beginning is not excluded and therefore it should be allowed.
If you have doubts, let's talk about this.

Reply:
Btw I have simplified the regex (without changing its outcome) to have it a little bit shorter.

Reply:
When I try to add whitespace to a filename in windows after the last proper character, it gets automatically removed. When I do the same at the front the same happens. And yes, the regex does not reflect that.

Reply:
Ahhh ok your first comment meant that it is not allowed as well on WINDOWS.
I thought you mean in the regex.

But then I should explicitly forbid a whitespace at the beginning via the Regex, right?

Reply:
Yup

Reply:
Ok pls check my last commit.  It should forbid a whitespace at the beginning now and also mention that in the comment.

Reply:
That does forbid whitespaces at any position. What about `^[^<>:"|?/*\\.\s][^<>:"|?/*\\]{0,253}[^<>:"|?/*\\.\s]$`?
This would however enforce a filename with at least 2 symbols which is probably fine.

Reply:
That forbids a `.` at the beginning, and allows a total fo 255 characters.

Check out my last commit.  Is there any problem you see in the regex?

Reply:
For got the "."  at the end.  Another commit

Reply:
The `.` was an accident.

Reply:
I reduced the center block to max 253 symbols